cluster/ci/config/prow/openshift/jenkins-config-updater/config-ci.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: jenkins-config-updater
+data:
+  update.yaml: |-
+    targets:
+    - cluster/ci/config/prow/openshift/jenkins-config-updater/deployment.yaml
+    - cluster/ci/config/prow/openshift/jenkins-config-updater/build.yaml
+    - cluster/ci/config/prow/openshift/jenkins-config-updater/config-ci.yaml
+    - projects/origin-release/pipeline.yaml
+    - projects/prometheus/prometheus.yaml
+    - projects/prometheus/node-exporter.yaml
+    - projects/kube-state-metrics/pipeline.yaml
+    - projects/gcsweb/pipeline.yaml
+    - projects/oauth-proxy/pipeline.yaml
+    - cluster/ci/config/submit-queue/submit_queue.yaml
+    - cluster/ci/config/submit-queue/submit_queue_openshift_ansible.yaml
+    - cluster/ci/config/submit-queue/submit_queue_origin_aggregated_logging.yaml
+    - cluster/ci/config/submit-queue/submit_queue_origin_web_console.yaml
+    - cluster/ci/config/prow/openshift/deck_rbac.yaml
+    - cluster/ci/config/prow/openshift/hook_rbac.yaml
+    - cluster/ci/config/prow/openshift/horologium_rbac.yaml
+    - cluster/ci/config/prow/openshift/jenkins_operator_rbac.yaml
+    - cluster/ci/config/prow/openshift/plank_rbac.yaml
+    - cluster/ci/config/prow/openshift/sinker_rbac.yaml
+    - cluster/ci/config/prow/openshift/splice_rbac.yaml
+    - cluster/ci/config/prow/openshift/deck.yaml
```

Comment:
None of the deployments is working because of https://github.com/openshift/origin/issues/17404

Reply:
>:(